docs/vscode-extension/envchecker-help.md

```diff
@@ -89,6 +89,19 @@ Please refer to [the guide](https://docs.microsoft.com/dotnet/core/tools/sdk-err
 
 The `global.json` file is not required, so if it doesn't contain anything other than the sdk node, you can delete the whole file.
 
+## Teams Toolkit Validation Settings
+
+There're two settings to turn on/off .NET SDK and Node.js validation, and both of them are enabled by default. You are able to uncheck the box if you don't need the dependencies validation and would like to handle it by yourself.
+
+To open your user and workspace settings, use the following VS Code menu command:
```

Comment:
VSCode -> Visual Studio Code
There're -> There are
don't need -> do not need (similar to the previous changes from Tim)

Reply:
![image](https://user-images.githubusercontent.com/15644078/118105761-619b2a00-b40f-11eb-8630-876a298bab84.png)
VS Code is copied from the official documentation 😂: https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings

Reply:
Updated to keep consistent with previous updates